app.js

```diff
@@ -251,9 +251,9 @@ function handleKeyUp (event) {
 
 function moveShipLeft() {
   if (airfighter.x > 0) {
-    airfighter.x -= 2;
+    airfighter.x -= 3;
```

Comment:
This Pull Request is about player information, I think it should not change airfighter movement speed